kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -103,7 +103,7 @@
             :title="$tr('bookmarks')"
             :link="getBookmarksLink()"
             :kind="$tr('bookmark')"
-            :description="'0 resources'"
+            :description="this.bookmarksCount + $tr('resources')"
```

Comment:
We cannot use string concatenation like this to construct user-facing strings, because in some languages the ordering of the words might need to be reversed.

Instead, always inject values into the string as variables using ICU syntax:
https://kolibri-dev.readthedocs.io/en/develop/i18n.html#icu-message-syntax